### PR TITLE
First version of Indicators Filters

### PR DESCRIPTION
**Pull request by cazapatamar**
First versión of Indicators Filters

### CODE REVIEW

File: src/IndicatorsDash.jsx [hunk 1]
```diff
@@ -56,7 +57,49 @@ class IndicatorsDash extends React.Component {
...
+      const tabs = {
+        'all': 'Todas',
+        'riesgo': 'Riesgo Biodiversidad',
+        'costo': 'Costo de Compensación',
+        'oportunidad' :'Oportunidad',
+        'monitoreo' :'Monitoreo',
+        'bioma' :'Bioma'
+      };
+
+     console.log('activeTab: ' + activeTab);
```
Comment (alegnaaived) [inline src/IndicatorsDash.jsx]: Please remove all console.log lines before pushing the code to the repository

File: src/IndicatorsDash.jsx [hunk 2]
```diff
@@ -56,7 +57,49 @@ class IndicatorsDash extends React.Component {
...
+                }
+              return(
+                      <div className={validClass} key={item.id}>
+                         <h2>{item.name}</h2>
+                         <h3>{item.class}</h3>
+                      </div>
+                    )
+            }
+          )}
+          {console.log(data)}
```
Comment (alegnaaived) [inline src/IndicatorsDash.jsx]: Same about console.log

**Review by alegnaaived — COMMENTED** [review]
Looking for only push functional code, please avoid console.log and comments in future commits. Everything else looks good (y)

File: src/IndicatorsDash.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/IndicatorsDash.jsx]: Done

File: src/IndicatorsDash.jsx (hunk 2 above)
Comment (cazapatamar) [reply src/IndicatorsDash.jsx]: Done

**Review by alegnaaived — APPROVED** [review]

**Pull request merged**